app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileRepo.kt

```diff
@@ -51,6 +60,7 @@ class PinnedTileRepo(private val applicationContext: Application) {
         if (screenshot != null) {
             PinnedTileScreenshotStore.saveAsync(applicationContext, newPinnedTile.id, screenshot)
         }
+        ++customTilesSize
```

Comment:
Unless I'm mistaken, there is a potential edge case here.

```
-> customTilesSize == 10, _pinnedTiles.value.size == 10
add 'www.example.com'
-> customTilesSize == 11, _pinnedTiles.value.size == 11
add 'www.example.com'
-> customTilesSize == 12, _pinnedTiles.value.size == 11
```

Reply:
Good catch